onadata/libs/serializers/password_reset_serializer.py

```diff
@@ -35,14 +35,13 @@ def get_password_reset_email(user, reset_url,
     """Creates the subject and email body for password reset email."""
     result = urlparse(reset_url)
     site_name = domain = result.hostname
-    encoded_username = urlsafe_base64_encode(
-        b(user.username.encode('utf-8'))).decode('utf-8')
+    encoded_username = urlsafe_base64_encode(b(user.username.encode('utf-8')))
```

Comment:
Change this because `urlsafe_base64_encode` now returns a string instead of a bytestring.